src/java.base/share/classes/sun/reflect/misc/ReflectUtil.java

```diff
@@ -129,7 +129,7 @@ private static void privateCheckPackageAccess(SecurityManager s, Class<?> clazz)
         }
```

Comment:
Pre-existing issue (so consider filing a follow-up) but this loop is redundant since `getPackageName` will already return the package name of the  component type.

Reply:
Seems to be true, I'll file another ticket for this